tests/integration/targets/iam_user/tasks/main.yml

```diff
@@ -34,7 +36,8 @@
 
   - name: create test user
     iam_user:
-      name: '{{ test_user }}'
+      name: "{{ test_user }}"
+      password: "{{ test_password }}"
```

Comment:
It would be better to leave these first few tests as they were.  We want to ensure as many of the possible code execution paths as possible are exercised.